lib/Db/QueueMapper.php

```diff
@@ -82,6 +83,27 @@ public function removeFileFromAllQueues(int $fileId) : void {
 		}
 	}
 
+	/**
+	 * @param string $model
+	 * @param \OCA\Recognize\Db\QueueFile $file
+	 * @return \OCA\Recognize\Db\QueueFile|null
+	 * @throws \OCP\AppFramework\Db\MultipleObjectsReturnedException
+	 * @throws \OCP\DB\Exception
+	 */
+	public function existsQueueItem(string $model, QueueFile $file) : ?QueueFile {
```

Comment:
Maybe just return a bool here as the name might suggest?

Reply:
Yeah, makes sense :)